less/plugins/breakpoints.js

```diff
@@ -91,14 +89,18 @@ functions.add('breakpoint-max', function ({ value: breakpointName }, breakpoints
 	const breakpointNames = Object.keys(breakpointsMap)
 	const breakpointIndex = breakpointNames.indexOf(breakpointName)
 
+	let nextBreakpoint
+
 	if (breakpointIndex === -1)
 		return new tree.Quoted('"')
 
 	// Maximum breakpoint width is null for the last breakpoint.
 	if ((breakpointIndex + 1) === breakpointNames.length)
 		return new tree.Quoted('"')
 
-	return new tree.Dimension((breakpointsMap[breakpointNames[breakpointIndex + 1]].value - 0.02), 'px')
+	nextBreakpoint = breakpointsMap[breakpointNames[breakpointIndex + 1]]
```

Comment:
Since this var isn’t used before this point, just declare it here (and remove the declaration above).
```suggestion
	const nextBreakpoint = breakpointsMap[breakpointNames[breakpointIndex + 1]]
```